homoalg.tex

```diff
@@ -1332,10 +1332,10 @@ \section{Garben}
 % I.5.5 (Keime und Halme)
 
 \begin{defn}
-  Sei $\Fais$ eine Garbe auf $X$. Der \emph{Halm} von $\Fais$ in $x \in X$ ist
+  Sei $\Fais$ eine Garbe auf $X$. Der \emph{Halm} o. die \emph{Faser} von $\Fais$ in $x \in X$ ist
```

Comment:
Halm heißt auf Englisch 'stalk'. In der aktuellen Version sind keine keine falschen Fasern mehr.